Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -92,7 +92,7 @@
   icon: { sprite: /Textures/Objects/Weapons/Melee/e_sword_double.rsi, state: icon }
   productEntity: EnergySwordDouble
   cost:
-    Telecrystal: 20 #(Originally 16)
+    Telecrystal: 16 #Originally Was 20 but no one bought it due to it's high cost
```

```suggestion
    Telecrystal: 16 # DeltaV - Originally 20 but no one bought it due to it's high cost.
```
Remember to cite that the change is a DeltaV change in the comment.